install/update_91_92.php

```diff
@@ -1012,6 +1012,151 @@ function update91to92() {
+      $query = "CREATE TABLE IF NOT EXISTS `glpi_linetypes` (
+         `id` int(11) NOT NULL AUTO_INCREMENT,
+         `name` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
+         `comment` text COLLATE utf8_unicode_ci,
+         `date_mod` datetime DEFAULT NULL,
+         `date_creation` datetime DEFAULT NULL,
+         PRIMARY KEY (`id`),
+         KEY `name` (`name`),
+         KEY `date_mod` (`date_mod`),
+         KEY `date_creation` (`date_creation`)
+         ) ENGINE=MyISAM  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+      $DB->queryOrDie($query, "9.2 add table glpi_linetypes");
+   }
+
+   if (!TableExists('glpi_lines')) {
+      $query = "CREATE TABLE `glpi_lines` (
+            `id`                   INT(11) NOT NULL auto_increment,
+            `name`                 VARCHAR(255) NOT NULL DEFAULT '',
+            `entities_id`          INT(11) NOT NULL DEFAULT 0,
+            `is_recursive`         TINYINT(1) NOT NULL DEFAULT 0,
+            `is_deleted`           TINYINT(1) NOT NULL DEFAULT 0,
+            `caller_num`           VARCHAR(255) NOT NULL DEFAULT '',
+            `caller_name`          VARCHAR(255) NOT NULL DEFAULT '',
+            `users_id`             INT(11) NOT NULL DEFAULT 0,
+            `groups_id`            INT(11) NOT NULL DEFAULT 0,
+            `lineoperators_id`     INT(11) NOT NULL DEFAULT 0,
+            `locations_id`         INT(11) NOT NULL DEFAULT '0',
+            `states_id`            INT(11) NOT NULL DEFAULT '0',
+            `linetypes_id`         INT(11) NOT NULL DEFAULT '0',
+            `date_creation`        DATETIME DEFAULT NULL,
+            `date_mod`             DATETIME DEFAULT NULL,
+            `comment`              TEXT DEFAULT NULL,
+            PRIMARY KEY            (`id`),
+            KEY `entities_id`      (`entities_id`),
+            KEY `is_recursive`     (`is_recursive`),
+            KEY `users_id`         (`users_id`),
+            KEY `lineoperators_id` (`lineoperators_id`)
+            ) ENGINE=MyISAM  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci";
+      $DB->queryOrDie($query, "9.2 add table glpi_lines");
+   }
+
+   if (!TableExists('glpi_devicesimcardtypes')) {
+      $query = "CREATE TABLE IF NOT EXISTS `glpi_devicesimcardtypes` (
+                  `id` int(11) NOT NULL AUTO_INCREMENT,
+                  `name` varchar(255) NOT NULL DEFAULT '',
+                  `comment` text COLLATE utf8_unicode_ci,
+                  `date_mod` datetime DEFAULT NULL,
+                  `date_creation` datetime DEFAULT NULL,
+                  PRIMARY KEY (`id`),
+                  KEY `name` (`name`),
+                  KEY `date_mod` (`date_mod`),
+                  KEY `date_creation` (`date_creation`)
+                ) ENGINE=MyISAM  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;";
+      $DB->queryOrDie($query, "9.2 add table glpi_devicesimcardtypes");
+   }
+
+   $DB->queryOrDie("INSERT INTO `glpi_devicesimcardtypes` VALUES ('1','Full SIM',NULL,NULL,NULL)");
```

Comment:
Cannot be run twice: "  SQL: INSERT INTO `glpi_devicesimcardtypes` VALUES (\'1\',\'Full SIM\',NULL,NULL,NULL)
  Error: Duplicate entry '1' for key 'PRIMARY'"